egs2/totonac/asr1/RESULTS.md

```diff
@@ -0,0 +1,32 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Fri Nov 12 02:34:12 EST 2021`
+- python version: `3.9.7 (default, Sep 16 2021, 13:09:58)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.4a1`
+- pytorch version: `pytorch 1.9.0`
```

Comment:
Can you upload a model?

Reply:
Sure, I can submit a transformer model now but a bit more time to run for feature fusion, cause Dan changed the interface, so my current pretrained model may not fit into the release code.

Reply:
I can upload a model with the new config later this week if you want @ftshijt

Reply:
Many thanks, that would be very helpful! If you want, you can use my prepared data folder to accelerate the procedure.

Reply:
I will dm you.